src/catalog/products.md

```diff
@@ -40,6 +40,7 @@ _Products Grid_
 |SKU|The unique Stock Keeping Unit that is assigned to the product.|
 |Price|The unit price of the product.|
 |Quantity|The quantity that is currently in stock.|
+|Salable Quantity|The quantity already sold.|
```

Comment:
I believe this is the quantity that is available for sale.

Reply:
@leslietilling your suggestion really makes sense, thank you ☺️